app/Repositories/CacheRepository.php

```diff
@@ -7,6 +7,14 @@
 class CacheRepository
 {
     /**
+     * Instantiate a new CacheRepository.
+     */
+    public function __construct()
+    {
+        $this->prefix = 'campaign';
```

Comment:
I don't think we want to hardcode this because we also [store user objects](https://github.com/DoSomething/rogue/blob/eb6d59a920611f5d022574660c0920182a55ebff/app/Services/Registrar.php#L16) in cache. The constructor could take in a `$prefix` and then set it when you instantiate it. 

So for example, here: 

Registrar.php
```
    public function __construct(Northstar $northstar, CacheRepository $cache)
    {
        $this->northstar = $northstar;
        $this->cache = $cache;
    }
```
You could do 

```    
public function __construct(Northstar $northstar)
    {
        $this->northstar = $northstar;
        $this->cache = new CacheRepository('user');
    }
```